tests/env.yml

```diff
@@ -2,7 +2,7 @@ name: test
 channels:
   - conda-forge
 dependencies:
-  - python=3.7
+  - python>=3.7
```

Comment:
Changed this because it somehow installed some pretty old `mamba=0.1.2=py37h*` build in the CI run before.